bin/automated

```diff
@@ -6,8 +6,6 @@
 set -e
 
 # Variables
-SOURCE_BRANCH="master"
-ORIGIN_URL=`git config --get remote.origin.url`
 ORIGIN_CREDENTIALS=${ORIGIN_URL/\/\/github.com/\/\/$GH_TOKEN@github.com}
 COMMIT_MESSAGE=$(git log -1 --pretty=%B)
 TALKS="infra-2017 machine-learning-2017 front-2017"
```

Comment:
1. No need to do it right away. If you don't find it belonging to this PR I can always open an issue if you prefer or if it's needed.
2. I see that `create-talk` modifies in place this script when a new talk is created... Most likely automated would benefit from a comment at some point as this is not standard practice...